libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/SFPushNotificationManager.m

```diff
@@ -152,6 +152,10 @@ - (BOOL)registerSalesforceNotificationsWithCompletionBlock:(SFUserAccount *)user
     if (rsaPublicKey) {
         bodyDict[@"RsaPublicKey"] = rsaPublicKey;
     }
+    if ([[SFRestAPI sharedInstance].apiVersion compare:@"v61.0"] >= 0) {
```

Comment:
I was going to make these constants but the rest of the body doesn’t use them so I thought this was more readable together, can change if anyone has a preference